.travis.yml

```diff
@@ -289,6 +292,7 @@ matrix:
         - mkdir -p build.ms && cd build.ms
         - cmake -G Ninja ${CMAKE_EXTRA_ARGS} -DCMAKE_BUILD_TYPE=${BLD_CONFIG} ..
         - travis_wait ${MAX_TIME_MIN} cmake --build . --parallel --verbose
+        - cat ${NIH_CACHE_ROOT}/ninja/MSVC_19.16.27034.0/Debug/src/grpc_src-stamp/*.log
```

Comment:
I think you should remove this line for now (we don't want it to be merged)

Reply:
removed